docs/integrations/azure-aks/k8s.mdx

```diff
@@ -66,46 +66,16 @@ Start by creating a new managed Kubernetes cluster in AKS. If you already have o
 
 ## **Step 2**: Install the ngrok Kubernetes Operator {#install-the-ngrok-kubernetes-ingress-controller}
 
-We recommend installing the [ngrok Kubernetes Ingress
-Controller](https://github.com/ngrok/ngrok-operator) before your end-user apps to separate the steps around infrastructure vs. app configuration.
+We recommend reserving a domain for ingress if you don't have one already.
```

Comment:
Yeah I think a step like this might be better off in the prerequisites section
1. Reserve a Domain - maybe give them the steps you've got here for doing that
2. Install Kubernetes Operator - link them to the guide

Reply:
Maybe we could replace that callout that says "This how-to requires:" with a `## Prerequisites` or `## Before you start` section